### DISCUSSION TITLE
Integrate known sign-posted hiking routes into the app

### DISCUSSION THREAD

**Discussion by RangoMcRoony** [organicmaps/organicmaps#7368] [Ideas]
Please see [https://hiking.waymarkedtrails.org/#help-about](https://hiking.waymarkedtrails.org/#help-about) for reference.

I've noticed many hiking and navigation apps use known sign-posted hiking routes as an optional overlay to their maps. I'm fairly certain a similar system is used by apps like FATMAP and Komoot as both have paths identical to the above link.

This is also mentioned under the walking routes section of the open street map wiki.

I think this would be particularly advantageous as if this is implemented it will be one of the only few apps that allow both navigation planning an access to hiking routes without a paywall and will likely give this app a huge advantage in the hiking community.

I'm not a developer so I'm not even sure if this is within the realm of possibility but if it is I genuinely believe this will be a game-changer.

I've also attached screenshots from Waymarked trails and Fatmap to show they are identical in terms of hiking route overlay
<img width="1260" alt="waymarked trails" src="https://github.com/organicmaps/organicmaps/assets/69477684/2593f305-40c9-4f0f-b62d-7b0211933f9d">.

An option to automatically route to existing waymarked trails would also be hugely advantageous.

There was a similar suggestion about using waymarkedtrails for cycling #6348 so I'm not sure if this is a repeat or not since this post is more focused on hiking trails.

Really amazing project and thanks for your amazing work :)
<img width="789" alt="fatmap" src="https://github.com/organicmaps/organicmaps/assets/69477684/95d77d83-a1d0-4ad6-a94f-d91a343dbd30">

=

**Comment by biodranik** [comment]
Is it #6218 ?

**Reply by RangoMcRoony** [reply]
I'm not entirely sure but it may well be, yes
